non-pinned.json

```diff
@@ -63,6 +63,10 @@
       "updateTypes": ["patch"],
       "dependencyDashboardApproval": true,
       "description": "View patch updates on the dashboard for Cargo, GitHub Actions, NPM and NuGet"
+    },
+    {
+      "matchUpdateTypes": ["major"],
+      "addLabels": ["major-update"]
```

Comment:
💭 Do we have formatting standards for labels?

Reply:
Not that I'm aware of.  I did try to follow the standard format that I saw in `clients`, for example:
<img width="650" height="464" alt="image" src="https://github.com/user-attachments/assets/5c34c403-d8fb-427c-bf57-9ae1bbc5ed53" />